### PR TITLE
[13.0][MIG] stock_split_picking: Migration to 13.0

### PR DESCRIPTION
**Pull request by xavierpiernas**
Standard migration

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍 . Maybe you can add yourself to contributors, if you want.

**Review by Hafdal-la — APPROVED** [review]

File: stock_split_picking/README.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fstock--logistics--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/stock-logistics-workflow/tree/12.0/stock_split_picking
```
Comment (CarlosRoca13) [inline stock_split_picking/README.rst:17]:
```suggestion
    :target: https://github.com/OCA/stock-logistics-workflow/tree/13.0/stock_split_picking
```
You have to generate the readme for the 13.0 version

File: stock_split_picking/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+* Nicolas Bessi <nicolas.bessi@camptocamp.com>
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
+* Yannick Vaucher <yannick.vaucher@camptocamp.com>
+* Vicent Cubells <vicent.cubells@tecnativa.com>
+* Julien Coux <julien.coux@camptocamp.com>
+* Andrius Preimantas <andrius@versada.eu>
```
Comment (CarlosRoca13) [inline stock_split_picking/readme/CONTRIBUTORS.rst:6]:
```suggestion
* Andrius Preimantas <andrius@versada.eu>
* Your name and surname <youremail if u want>
```
I think you would like to be a contributor

**Review by CarlosRoca13 — COMMENTED** [review]
It's important to generate the readme following the instructions that we find on the readme of https://github.com/OCA/maintainer-tools . 

I hope I have been helpful

**Comment by rousseldenis** [comment]
> It's important to generate the readme following the instructions that we find on the readme of https://github.com/OCA/maintainer-tools .
> 
> I hope I have been helpful

As readme fragments folder has been populated, the README should be generated automatically with oca-bot after merge

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by CarlosRoca13 — APPROVED** [review]
@rousseldenis Okey, thanks for the info

So I aprove too :+1:

**Pull request merged**